openshift-hack/e2e/annotate/rules.go

```diff
@@ -15,8 +15,6 @@ var (
 			`\[Feature:ClusterTrustBundle\]`,
 			`\[Feature:SELinuxMount\]`,
 			`\[FeatureGate:SELinuxMount\]`,
-			`\[Feature:UserNamespacesPodSecurityStandards\]`,
-			`\[Feature:UserNamespacesSupport\]`, // disabled Beta
```

Comment:
We need to remove these from `openshift-hack/cmd/k8s-tests-ext/disabled_tests.go` as well